lib/bolt/cli.rb

```diff
@@ -107,6 +149,16 @@ def parse_command
         # help text
         options[:subcommand] = nil unless COMMANDS.include?(options[:subcommand])
 
+        if Bolt::Util.first_run? && Bolt::Config.user_path
+          FileUtils.mkdir_p(Bolt::Util.first_runs_free.dirname)
+          FileUtils.touch(Bolt::Util.first_runs_free)
+
+          if options[:subcommand].nil? && $stdout.isatty
```

Comment:
Could `$stdout.isatty` be part of `Bolt::Util.first_run?` ? Not sure if there's a need to create the first_runs_free file if stdin isn't a tty.

Reply:
I think there might be other cases where we want to know if this is the first time Bolt has been run or not that won't rely on whether stdout is a tty, so I'd like to leave that here as it's specific to the welcome message.